package.json

```diff
@@ -47,8 +47,12 @@
     "@types/update-notifier": "5.1.0",
     "@types/uuid": "8.3.4",
     "@types/yargs": "17.0.10",
+    "@typescript-eslint/eslint-plugin": "^5.25.0",
+    "@typescript-eslint/parser": "^5.25.0",
+    "eslint": "^8.16.0",
+    "eslint-plugin-import": "^2.26.0",
     "prettier": "2.6.2",
-    "typescript": "4.6.4"
+    "typescript": "^4.6.4"
```

Comment:
can we get rid of the `^` on these? (is there a lint rule for that?)

Reply:
Figured it out -- take a look at the newer commits.